http/oidc/src/main/java/org/wildfly/security/http/oidc/OidcHttpFacade.java

```diff
@@ -180,13 +181,17 @@ OidcTokenStore getTokenStore() {
     }
 
     OidcClientConfiguration getOidcClientConfiguration() {
-        return oidcClientContext.resolveDeployment(this);
+        if (this.oidcClientConfiguration == null) {
+            this.oidcClientConfiguration = oidcClientContext.resolveDeployment(this);
+        }
+        return oidcClientConfiguration;
```

Comment:
I think these changes and the ones below shouldn't be needed anymore.

Reply:
fixed this. Thanks!